docs/api.md

```diff
@@ -0,0 +1,49 @@
+# Renovate API
+
+Renovate CE/EE exposes a REST API that can be used to interact programmatically with Renovate.
+
+## Enabling and Authentication
+
+The API can be enabled by setting the `MEND_RNV_ADMIN_API_ENABLED` environment variable to `true`.
+You must also configure an API secret by setting the `MEND_RNV_SERVER_API_SECRET` variable.
+
+Authentication is done via HTTP Auth, using the API secret as the password.
+For example if the secret is `renovateapi` then you would authenticate with:
+
+```
+  Authorization: renovateapi
+```
+
+## Endpoints
+
+### Health
+
+`GET /health`
+
+Returns 200 if API is healthy.
+
+### Queue status
+
+`GET /api/job/queue`
+
+Returns the current status of the job queue, including number of pending jobs.
+
+`GET /api/task/queue`
```

Comment:
Nice, this is what used to be available from I think the `/status` endpoint on Renovate On Prem, right? Glad to see it's now behind an authN check 👏🏽 (my org was concerned that it may expose information about private repo names to the public Internet, so we restricted traffic at the Load Balancer)

Reply:
Yes, that was the intention. I don't think we exposed "task" information as that's a new internal concept part of the CE rewrite, but we exposed _job_ information

Reply:
Health endpoint is not authenticated and responds with a status code only.
All other endpoints are authenticated tho